counter/styles.css

```
@@ -81,6 +81,18 @@ body {
   padding: 0.50rem;
 }
 
+.counter__value.counter__value-zero {
+  color: gray;
+}
+
+.counter__value.counter__value-positive {
+  color: green;
+}
+
+.counter__value.counter__value-negative {
+  color: tomato;
+}
```

давай придерживаться классической БЭМ-нотации для модификаторов:
```suggestion
.counter__value.counter__value--zero {
  color: gray;
}

.counter__value.counter__value--positive {
  color: green;
}

.counter__value.counter__value--negative {
  color: tomato;
}
```